### PR TITLE
GSK-1035 Add language detection in scan

### PR DESCRIPTION
**Pull request by mathieuroques**
I used detect_lang and I support english and french

### CODE REVIEW

**Comment by mattbit** [comment]
@mathieuroques could you fix the conflicts with the base branch?

**Comment by mathieuroques** [comment]
Ok let me handle this

**Comment by mathieuroques** [comment]
@mattbit the conflicts have been resolved

**Comment by mathieuroques** [comment]
Just fixed conflicts and the failing test

File: python-client/giskard/scanner/robustness/text_transformations.py [hunk 1]
```diff
@@ -116,10 +117,11 @@ class TextGenderTransformation(TextTransformation):
     name = "Switch gender"
 
     def make_perturbation(self, x):
+        language = detect_langs(x)[0].lang
```
Comment (mattbit) [inline python-client/giskard/scanner/robustness/text_transformations.py:120]: This should be calculated once in dataset text metadata provider (search `TextMetadataProvider`), and than accessed at the dataset level with `dataset.column_meta["my_textual_column", "text"].language`.

**Review by mattbit — CHANGES_REQUESTED** [review]

File: python-client/giskard/scanner/robustness/entity_swap.py
Comment (mattbit) [inline python-client/giskard/scanner/robustness/entity_swap.py:1]: Can we put this into some csv or similar and load them only if needed?

**Pull request merged**